Automattic/jetpack
# Jetpack Components: Add Toggle

## Proposed changes:

* Adds a `<ToggleControl />` component to `@automattic/jetpack-components`.

### Other information:


## Jetpack product discussion



TTzDk3dzct551hYWRXPFa3-fi-2394%3A28057

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Check out this branch
* Run `jetpack install -r`
* Run `cd projects/js-packages/storybook`
* Run `pnpm run storybook:dev`
* Review the "Toggle Control" component stories and compare them to the Figma designs.

## Screenshots

<img width="1132" alt="Screenshot 2023-04-18 at 4 42 41 PM" src="https://user-images.githubusercontent.com/10933065/232920787-055c1f66-8a25-4878-9c29-6d6cd2cdcfc1.png">
